website_multi_theme/README.rst

```diff
@@ -125,6 +125,14 @@ Known issues / Roadmap
   become the base for all those supported, which can easily lead to weird
   situations and errors.
 
+Required bug patch
+------------------
+
+You need to include this code in your Odoo instance for the addon to work.
+No workaround possible for this one:
```

Comment:
You probably mean 

    No workaround (without monkey patches) possible for this one:

don't you?)